extension/src/plots/paths/collect.ts

```diff
@@ -196,6 +197,15 @@ const collectOrderedPath = (
   return acc
 }
 
+const fun = (path: string): string[] => {
+  if (!path.includes('::')) {
+    return getPathArray(path)
+  }
+
+  const [dvcYamlPath, plotPath] = path.split('::')
+  return [dvcYamlPath, ...getPathArray(plotPath)]
```

Comment:
The first entry will always be the path to the dvc.yaml probably needs another field as we use the information in the tree ([here](https://github.com/iterative/vscode-dvc/pull/4527/files#diff-69f2a655cd9dd9767fb87d0d4f5cfcaabd011d1a8212d0bb0b45691d9c4423edR64))